packages/client-beta/src/views/MenuBar/components/TurnOffWidget/TurnOffWidget.tsx

```diff
@@ -0,0 +1,13 @@
+import powerOffIcon from '../../../../icons/power-off-solid.svg';
+import styles from './TurnOffWidget.module.css';
+import { UserAccountService } from '../../../../services/user-account-service';
+
+function TurnOffWidget() {
+  return (
+    <button class={styles.buttonIcon} onClick={UserAccountService.userLogout}>
+      <img class={styles.icon} src={powerOffIcon} alt="Uśpij. Nie wyłączaj!" />
```

Comment:
To pierdoła, ale jak przestawiliśmy język na en to pewnie alty też warto by było po angielsku